src/sqlfluff/rules/aliasing/__init__.py

```diff
@@ -2,16 +2,20 @@
 
 from sqlfluff.core.plugin import hookimpl
 
-from sqlfluff.rules.aliasing.AL01 import Rule_AL01
-from sqlfluff.rules.aliasing.AL02 import Rule_AL02
-from sqlfluff.rules.aliasing.AL03 import Rule_AL03
-from sqlfluff.rules.aliasing.AL04 import Rule_AL04
-from sqlfluff.rules.aliasing.AL05 import Rule_AL05
-from sqlfluff.rules.aliasing.AL06 import Rule_AL06
-from sqlfluff.rules.aliasing.AL07 import Rule_AL07
-
 
 @hookimpl
 def get_rules():
-    """Get plugin rules."""
+    """Get plugin rules.
+
+    NOTE: Rules are imported only on fetch to manage import times
+    when rules aren't used.
+    """
+    from sqlfluff.rules.aliasing.AL01 import Rule_AL01
```

Comment:
can we not do `from sqlfluff.rules.aliasing import Rule_AL01, Rule_AL02...` here?

Reply:
Unfortunately not because they're all in their own source files. `Rule_AL01` comes from `AL01.py` and `Rule_AL02` comes from `AL02.py` etc...